asserts/system_user.go

```diff
@@ -216,6 +246,30 @@ func checkHashedPassword(headers map[string]interface{}, name string) (string, e
 	return pw, nil
 }
 
+func parseSystemUserExpiration(headers map[string]interface{}, until time.Time) (string, error) {
```

Comment:
parseShadowLine is an outlier here, this should also be called check to be consistent

Reply:
Fixed!